test/gulp.js

```diff
@@ -200,7 +201,9 @@ describe('gulp-google-closure-compiler', function() {
             platform
           });
 
-          stream.pipe(assert.length(2))
+          stream
+              .pipe(streamFilter(['**', '!**/$weak$.js']))
```

Comment:
Please add a comment explaining what this `'$weak$.js'` file is and why it must be filtered out.

Reply:
Done